app/src/main/java/org/oppia/app/home/HomeFragmentController.kt

```diff
@@ -34,4 +50,19 @@ class HomeFragmentController @Inject constructor(
   private fun getUserAppHistoryViewModel(): UserAppHistoryViewModel {
     return viewModelProvider.getForFragment(fragment, UserAppHistoryViewModel::class.java)
   }
+
+  fun goToExplorationButtonClicked(v: View) {
+    explorationDataController.startPlayingExploration(
+      TEST_EXPLORATION_ID
+    ).observe(fragment, Observer<AsyncResult<Any?>> { result ->
+      when {
+        result.isPending() -> logger.d("HomeFragmentController", "Loading exploration")
+        result.isFailure() -> logger.e("HomeFragmentController", "Failed to load exploration", result.getErrorOrNull()!!)
```

Comment:
Is the ``!!`` operator needed here? I think it's fine to pass null into the logger (though realistically it will never be null per AsyncResult preventing that from happening).

Reply:
Actually it is giving type mismatch error and therefore we will need this.

`Type mismatch.`
`Required: Throwable`
`Found: Throwable?`